src/test/java/TestJunit.java

```diff
@@ -0,0 +1,46 @@
+import org.testng.annotations.Test;
+import static org.testng.Assert.assertEquals;
+
+public class TestJunit {
+
+    @Test
+    public void testAddMethod() {
+        CustomList myList = new CustomList(16);
+        myList.add(10);
+        assertEquals(1, myList.getSize());
+        myList.add(10);
+        assertEquals(2, myList.getSize());
+        myList.add(10);
+        assertEquals(3, myList.getSize());
+    }
+
+    @Test
+    public void testConstructor() {
+        CustomList myList = new CustomList(16);
+        assertEquals(16, myList.getCapacity());
```

Comment:
just check, sometimes, it makes sense to use different methods, like assertTrue, and others.